src/sentry/static/sentry/app/views/alerts/wizard/index.tsx

```diff
@@ -99,25 +101,31 @@ class AlertWizard extends React.Component<Props, State> {
             <StyledPageHeader>
               <PageHeading>{t('What do you want to alert on?')}</PageHeading>
             </StyledPageHeader>
+            <Heading>{t('Errors')}</Heading>
             <WizardBody>
               <WizardOptions>
-                {AlertWizardOptions.map(({categoryHeading, options}) => (
+                {AlertWizardOptions.map(({categoryHeading, options}, i) => (
                   <OptionsWrapper key={categoryHeading}>
-                    <Heading>{categoryHeading}</Heading>
-                    <RadioGroup
-                      choices={options.map(alertType => [
-                        alertType,
-                        AlertWizardAlertNames[alertType],
-                      ])}
+                    {!!i && <Heading>{categoryHeading}</Heading>}
```

Comment:
is this checking that it isn't the first row? can we check i > 0

Reply:
yup will do